data/seeds/006_Children.js

```diff
@@ -107,7 +108,7 @@ exports.seed = function (knex) {
   return knex('Children').insert(children)
   .then(function(){
     return knex('Children').insert([
-      { Name: `${faker.name.lastName("McVirtual")} (Cohort3)`,
+      { Name: `${faker.name.firstName()} "McVirtual"(Cohort3)`,
```

Comment:
These changes look good, but instead of adding the child here, I think it would be better to put this in a variable like "children5" or something

Reply:
Why would I do this? I've already  made multiple changes  and meeting the required changes that needed to be done. You're more than welcome to make any further changes.

Reply:
The reason is that the other children have their own variable as well. It would match the rest of the file and keep the code more organized. If you want to be done working on this ticket, I'll try to finish it up.

Reply:
Ok gotcha, I'll try to get it done.